internal/cluster/statemachine.go

```diff
@@ -34,6 +35,7 @@ func NewClusterStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		SourceStates: []stateswitch.State{
 			stateswitch.State(models.ClusterStatusPreparingForInstallation),
 			stateswitch.State(models.ClusterStatusInstalling),
+			stateswitch.State(models.ClusterStatusInstallingPendingUserAction),
```

Comment:
Curios, how this can happen?

Reply:
How can we get from Installing to Insufficient as well?

Reply:
+1

Reply:
during installation if some hosts are in error the ui exposes reset option directly.

Reply:
@avishayt we can get to it by skipping the cancellation and resetting the cluster from installing

Reply:
like a hacky tool to try to fix installation?

Reply:
yes, and also for cases that hosts in error, ui will expose reset button directly.